docs/articles/preamble.md

```diff
@@ -9,7 +9,7 @@ Before attempting to write a Discord bot, you should be familiar with the concep
 If you're brand new to C#, or programming in general, this library may prove difficult for you to use.</br>
 Fortunately, there are resources that can help you get started with the language! 
 
-An exellent tutorial series to go through would be [C# Fundamentals for Absolute Beginners](https://channel9.msdn.com/Series/C-Fundamentals-for-Absolute-Beginners) by Bob Tabor.
+An excellent tutorial series to go through would be [C# Fundamentals for Absolute Beginners](https://channel9.msdn.com/Series/C-Fundamentals-for-Absolute-Beginners) by Bob Tabor.
```

Comment:
My only suggestion that I might add I'd that these videos were obsoleted by new videos by this guy and other MSDN tutorials so we might consider the other ones as well. I'm not at my PC to get them right now unfortunately. If I'm reminded I will grab that later.

Reply:
Adjusted the URL to Bob Tabor's most recent C# Fundamentals for Absolute Beginners (2019). Thanks for the suggestion!